flake.nix

```diff
@@ -1,22 +1,21 @@
 {
   description = "Nix expressions for defining Replit development environments";
-  inputs.nixpkgs.url = "github:nixos/nixpkgs?rev=52e3e80afff4b16ccb7c52e9f0f5220552f03d04";
-  inputs.nixpkgs-unstable.url = "github:nixos/nixpkgs?rev=87f9156865ab09e3bde39aadb4131ae364ae704e";
-  inputs.prybar.url = "github:replit/prybar?rev=65f486534054665f1b333689417c39acd370d3a5";
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-23.05";
+  inputs.nixpkgs-unstable.url = "github:nixos/nixpkgs/nixpkgs-unstable";
```

Comment:
using `?rev` [no longer needed](https://replit.slack.com/archives/C03KS2B221W/p1686178911723559?thread_ts=1686154091.642739&cid=C03KS2B221W)

Reply:
I suspect we're still going to need it, but happy to try again!